snapd.go

```diff
@@ -249,13 +249,11 @@ func action(ctx *cli.Context) {
 	setMaxProcs(cfg.GoMaxProcs)
 
 	c := control.New(cfg.Control)
-
-	coreModules = []coreModule{}
-
-	coreModules = append(coreModules, c)
 	s := scheduler.New(cfg.Scheduler)
+	c.SetTaskScheduler(s)
 	s.SetMetricManager(c)
-	coreModules = append(coreModules, s)
+
+	coreModules = []coreModule{c, s}
```

Comment:
I need to talk to scheduler from control to update task deps on plugin load event. Is there any way to know that the new plugin was just loaded from scheduler point of view?